ddl/index.go

```diff
@@ -1119,27 +1119,27 @@ func (w *worker) updateReorgInfo(t table.PartitionedTable, reorg *reorgInfo) (bo
 		return false, errors.Trace(err)
 	}
 	log.Infof("[ddl-reorg] job %v update reorgInfo partition %d range [%d %d]", reorg.Job.ID, pid, start, end)
-	reorg.StartHandle, reorg.EndHandle, reorg.PartitionID = start, end, pid
+	reorg.StartHandle, reorg.EndHandle, reorg.PhysicalID = start, end, pid
 
 	// Write the reorg info to store so the whole reorganize process can recover from panic.
 	err = kv.RunInNewTxn(reorg.d.store, true, func(txn kv.Transaction) error {
-		return errors.Trace(reorg.UpdateReorgMeta(txn, reorg.StartHandle, reorg.EndHandle, reorg.PartitionID))
+		return errors.Trace(reorg.UpdateReorgMeta(txn, reorg.StartHandle, reorg.EndHandle, reorg.PhysicalID))
 	})
 	return false, errors.Trace(err)
 }
 
-// findNextPartitionID finds the next partition ID in the PartitionDefinition array.
-// Returns 0 if current partitionID is already the last one.
-func findNextPartitionID(partitionID int64, defs []model.PartitionDefinition) (int64, error) {
+// findNextPhysicalID finds the next partition ID in the PartitionDefinition array.
+// Returns 0 if current PhysicalID is already the last one.
+func findNextPhysicalID(physicalID int64, defs []model.PartitionDefinition) (int64, error) {
```

Comment:
I think this function should use `PartitionID`, the argument is PartitionDefinition.